gr-azure-software-radio/python/blob_sink.py

```diff
@@ -123,7 +141,7 @@ def create_blob(self):
         """
         try:
             self.blob_client.commit_block_list(block_list=[])
-        except az_exceptions.HttpResponseError as err:
+        except (az_exceptions.HttpResponseError, urllib3.exceptions.LocationParseError) as err:
```

Comment:
What is the location parse error? or how does it occur?

Reply:
If I remember right, that was getting triggered if you included a trailing '/' in the URL. I can add a test case that forces it to make it clear